### PR TITLE
feat(ask-sdk-core/util) Add new helper function to get user id

### PR DESCRIPTION
**Pull request by hideokamoto**
We can get user id from the SDK.

## Description
Add the `getUserId` function into the utils.
We can get the user id more easier.

## Motivation and Context
Now we can get the requested user's device id by the SDK.
But we want to get the user id too.

### Before

```typescript
const userId = handlerInput.requestEnvelope.context.System.user.userId 
```

Our code will be too long line :(

### After

```typescript
const userId = getUserId(handlerInput.requestEnvelope)
```

It is the simplest way to get the user ID :)

## Testing

To run unit test by gulp.

```bash
$ lerna bootstrap
$ lerna --scope ask-sdk-core run gulp -- test
  110 passing (76ms)


=============================== Coverage summary ===============================
Statements   : 100% ( 351/351 )
Branches     : 100% ( 88/88 )
Functions    : 96.26% ( 103/107 )
Lines        : 100% ( 330/330 )
================================================================================

[09:57:52] Finished 'test' after 1.76 s
lerna success run Ran npm script 'gulp' in 1 package in 2.9s:
lerna success - ask-sdk-core
```

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Docs(Add new document content)
- [ ] Translate Docs(Translate document content)

## Checklist

- [x] My code follows the code style of this project
- [x] My change requires a change to the documentation
- [x] I have updated the documentation accordingly
- [x] I have read the **README** document
- [x] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

## License

- [x] I confirm that this pull request can be released under the Apache 2 license

[issues]: https://github.com/alexa/alexa-skills-kit-sdk-for-nodejs/issues
[license]: http://aws.amazon.com/apache2.0/
[cla]: http://en.wikipedia.org/wiki/Contributor_License_Agreement

### CODE REVIEW

**Comment by kun432** [comment]
:+1:

File: ask-sdk-core/tst/util/RequestEnvelopeUtil.spec.ts [hunk 1]
```diff
@@ -43,7 +44,7 @@ describe('RequestEnvelopeUtils', () => {
             templateVersion : '1.0',
             markupVersion : '1.0',
         },
-    };
+    };requestEnvelope.context.System.user.userId = 'mockUserId'
```
Comment (ShenChen93) [inline ask-sdk-core/tst/util/RequestEnvelopeUtil.spec.ts]: Could you please write this one in a new line?

Comment (hideokamoto) [reply ask-sdk-core/tst/util/RequestEnvelopeUtil.spec.ts]: Thanks for the feedback!
I've fixed it :)

**Comment by ShenChen93** [comment]
Hi @hideokamoto 

The code looks good. Before I merge it in, could you please modify the commit name to "feat: add getUserId util function", In order to fit conventional commits structure.

Thanks,
Shen

**Comment by hideokamoto** [comment]
I've rebased my commits.

**Review by ShenChen93 — APPROVED** [review]

**Review by nikhilym — APPROVED** [review]
:shipit:

**Comment by ShenChen93** [comment]
Hi, @hideokamoto 

Thanks for the PR! Merging this in. It will be included in the next SDK release.

Regards,
Shen

**Pull request merged**